Vic2ToHoI4/Source/Configuration.cpp

```diff
@@ -162,7 +162,11 @@ Configuration::Factory::Factory()
 			 std::clamp(static_cast<float>(commonItems::singleDouble{theStream}.getDouble()), 0.0F, 100.0F) / 100.0F;
 		Log(LogLevel::Info) << "\tPercent of commanders: " << configuration->percentOfCommanders;
 	});
-	registerRegex(commonItems::catchallRegex, commonItems::ignoreItem);
+	registerKeyword("output_name", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString outputNameStr(theStream);
+		configuration->customOutputName = outputNameStr.getString();
```

Comment:
this can be shortened to:
configuration->customOutputName = commonItems::getString(theStream);

You can delete the line above (const commonItems::singleString outputNameStr(theStream);)

Reply:
I have tried that. It doesn't work, though.

Reply:
As mentioned on Discord, Vic2ToHoI4 is on an old version of the commonItems. For now, you can instead do
```configuration->customOutputName = commonItems::singleString(theStream).getString();```